Jint/Runtime/Interpreter/Expressions/JintObjectExpression.cs

```diff
@@ -107,7 +107,7 @@ private object BuildObjectFast()
             {
                 var objectProperty = _properties[i];
                 var valueExpression = _valueExpressions[i];
-                var propValue = valueExpression.GetValue();
+                var propValue = valueExpression.GetValue().Clone();
                 hasSymbols |= objectProperty.Key.IsSymbol;
```

Comment:
I don't see the full file, is this clone method used it for Arguments instance?

Reply:
Yes, it's now the same logic like in ConcatenatedString, a moment to materialize things as there's an assignment that might lead to moving out of original scope (function body). Now IMO cleaner and was able to remove the FunctionWasCalled from binding context (making it better for you).